### DISCUSSION TITLE
How do I contact support?

### DISCUSSION THREAD

**Discussion by Luctia** [getsentry/support#7] [Support] [answered]
Yes, I am aware that this isn't the right place for this. However, I'm trying to transwer ownership and to do that, I should contact support. The "contact support" button on [help.sentry.io](https://help.sentry.io) does nothing, Twitter messages are closed and there's no other way than GitHub discussions even listed, so here I am. What should I do?

**Comment by Luctia** [comment, marked as answer]
Right, so, the "contact support" button _does_ seem to work, just not on the homepage?

**Reply by chadwhitacre** [reply]
By homepage do you mean the issues page? I get a search bar.

<img width="826" alt="Screen Shot 2022-05-23 at 3 58 46 PM" src="https://user-images.githubusercontent.com/134455/169896440-bcc1d4cd-f6dd-4dfa-9591-04de34b36cd0.png">

**Reply by Luctia** [reply]
I'm talking about the button on [help.sentry.io](https://help.sentry.io). Under "All the help you could dream of", there's a button "contact support" that does nothing.

**Reply by chadwhitacre** [reply]
Ah, gotcha. Yeah if that reads "Contact support" then it's supposed to open up a Zendesk modal. Maybe you have an ad blocker or other browser plugin that is interfering?

<img width="530" alt="zendesk" src="https://user-images.githubusercontent.com/134455/170068910-1e596c5f-2848-4b6a-a7cd-e39ff5be2167.png">
